docs/platform/stackscripts-new-manager/index.md

```diff
@@ -0,0 +1,307 @@
+
+    ![Navigate to Create from StackScript.](stackscripts-navigate-to-create-from-stackscript.png "Navigate to Create from Stackscript.")
+
+3.  At this point you have the option of using a a Linode StackScript, Community StackScript, or one of your own.
+
+    [![StackScript selection options.](stackscripts-selection-screen-small.png "StackScript selection options.")](stackscripts-selection-screen.png)
+
+<!-- Alternatively, you can follow along with this video, which will show you how to deploy from a StackScript:
+
+{{< youtube XviHHyXVH20 >}} -->
+
+### Using a Community StackScript
+
+{{< note >}}
+There is no default logging when using a StackScript. Output is sent to the console.
+{{< /note >}}
+
+1.  In addition to the StackScripts created by Linode, any community member who creates a StackScript has the option to make it publicly available to other customers. As seen in the image below, you can use the search field to narrow your search to match your distribution and goal:
+
+    [![A search for 'wordpress.'](stackscripts-search-wordpress-small.png "A search for 'wordpress.'")](stackscripts-search-wordpress.png)
+
+2.  For this example we'll select the first search result. This brings us to a page where we can define the variables that will go into the StackScript. In this case, the MySQL root password and WordPress database name. The other options are standard whenever you deploy a new Linode.
+
+    ![StackScript options](stackscripts-wordpress-options.png "Stackscript options.")
+
+3.  Once you've adjusted the options, hit **Create**. You will be returned to the Linodes page, and your bell notifications will inform you when your Linode has been created.
+
+    ![Bell notification of new Linode creation.](stackscripts-linode-created-bell.png "Bell notification of a new Linode creation.")
+
+    ![The newly created Linode.](stackscripts-linode-created.png "The newly created Linode.")
+
+
+### Creating a New StackScript
+
+1.  To begin creating your own StackScript, navigate to the **StackScripts** page of the manager and click **Create New StackScript**.
+
+    [![Create new StackScripts button.](stackscripts-create-stackscript-small.png "Create New StackScripts button.")](stackscripts-create-stackscript.png)
+
+2.  Fill in the various sections:
+
+    -   *Label* - How the StackScript will be identified.
+    -   *Description* - a brief explanation of what it does.
+    -   *Target Images* - Select any and all available distributions that can run this StackScript.
+    -   *Script* - The body of the script itself.
+    -   *Revision Note* A brief account of the change made in this update of the StackScript. It will be set to `Initial import` by default, and you can feel free to leave or change it.
+
+    Below is an example of a new StackScript being written:
+
+    [![A StackScript being created.](stackscripts-new-stackscript-options-small.png "A new StackScript being created.")](stackscripts-new-stackscript-options.png)
```

Comment:
This form has changed a bit, the revision note is a single line rather than a textarea.